### DISCUSSION TITLE
incorrect fields in interlaced video

### DISCUSSION THREAD

**Discussion by dimar112q** [staxrip/staxrip#1028] [Ideas]
I discovered AviSynth filter called MSU Field Shift Fixer.
I was wondering if something like that is available in StaxRip, or if this is something that could be implemented in the future?
I'm assuming that the filter is 32-bit since the files are dated 2004.

More info here
Free AviSynth filter for fixing situation when different lines of source frame are placed in different output frames.
https://www.compression.ru/video/old_film_recover/field_shift_en.html

**Comment by Dendraspis** [comment]
I'm pretty sure the issues were fixed in the last 19 years. 😅

Nobody reported any problems with incorrect fields and on top 32bit filters are not supported anymore.

Did you experience any issues or did you request just in case it could be an issue for you in the future?

**Reply by dimar112q** [reply]
I just thought this was a really cool filter to fix field issues on certain badly interlaced videos. I have some DVDs with weird interlacing that create ghost images when deinterlaced and was thinking something like this could help.

**Reply by Dendraspis** [reply]
A valid question. Maybe it could help discuss that on our Discord server, but for that particular filter there is no way I can integrate it.
